recipes/pyagg/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "30a1c5519696ff868104cfe7d33c8ec52e8698a8be6e21a3b6eb2d0c168f28aa" %}
+
+package:
+  name: pyagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jwiggins/pyagg/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - cython
+    - freetype
```

Comment:
`Travis-CI` is trying to use `pkg-config`. You may add it here as a dependency (not sure if that will solve the error though :smile:).

To keep `pyagg` consistent with the rest of the stack in conda-forge you need to pin `freetype` to `freetype 2.6.*`. See https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies for the current pinning in `conda-forge`.